### ISSUE TITLE
Deck Tracker not responding

### ISSUE THREAD

**Issue by HrBak** [Epix37/Hearthstone-Deck-Tracker#1586]
Hello,

thanks a lot for the awesome work with the deck tracker!
Unfortunately it doesn't work anymore.
When i launch it, it gets stuck on the main windows, and then doesn't respond.
http://imgur.com/2u6Bj2r

I already tried clearing the config in AppData and deleting the whole folder from AppData, but the issue is still the same.

Any suggestions?

Best regards

**Comment by Epix37** [comment]
Try uninstalling and reinstalling .NET Framework 4.5.

**Comment by HrBak** [comment]
I'm running Windows 10, so It's running .Net 4.6 - even if i would, i cannot install .NET Framework 4.5 :/

**Comment by Epix37** [comment]
Well.. try reinstalling 4.6 then :).

**Comment by HrBak** [comment]
From what I can find, it is impossible reinstalling .NET Framework 4.6 - I think i tried everything.

**Comment by Epix37** [comment]
Oh, that might be true. Hm... 

Try redownloading and extracting it somewhere else (e.g. desktop) and see if that does anything. Maybe running as administrator. 

Very weird that HDT does not produce a crash message.

**Comment by HrBak** [comment]
Okay - so I tried extracting it on another drive, and on the desktop and the result is the exact same. Also tried running as administrator but still without any luck and the with the same behavior.
It is really wierd :(

**Comment by HrBak** [comment]
Okay, I've tried some stuff, and I got it working.
No matter where I extracted HDT it didn't work, so I tried reinstalling Windows 10, with my personal files saved, this "reinstalls" .NET Framework 4.6.
HDT then asked for the hearthstone path, and after I selected the usual path, the same error occured.
I then uninstalled hearthstone from my secondary drive and reinstalled it on my primary drive, deleted the config file from HDT and reset the hearthstone path, and now it is working again.

TLDR; I reinstalled Hearthstone to my primary drive, where Win 10 is installed, and now HDT is working.

Keep up the hard work!

**Comment by Epix37** [comment]
Huh intereesting. No idea why that would cause it, good to know though! Thank you for reporting back!
